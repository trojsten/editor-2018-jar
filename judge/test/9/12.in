VECTOR_INT:
lode
4
3
4
2
1
INT:
nepouzijestentointnadarmo
0
