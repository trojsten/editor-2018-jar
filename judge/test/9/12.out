{"lode": [1, 2, 4, 3]}